example-website/src/app/layout.tsx

```diff
@@ -17,6 +18,7 @@ export default function RootLayout({
   return (
     <html lang="en">
       <body className={inter.className}>{children}</body>
+      <Footer />
```

Comment:
This causes some errors to appear, I moved the <Footer /> to page.tsx (where <Header /> is) and it fixes it. Since I'm responsible for putting together the Home page, you can just merge this PR and I'll make the change in my PR.